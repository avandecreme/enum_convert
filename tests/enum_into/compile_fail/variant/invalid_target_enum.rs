use enum_convert::EnumInto;

#[derive(EnumInto)]
#[enum_into(Target)]
enum Source {
    #[enum_into(Target, NonExistent)] // Invalid target enum
    Unit,
}

enum Target {
    Unit,
}

fn main() {}
