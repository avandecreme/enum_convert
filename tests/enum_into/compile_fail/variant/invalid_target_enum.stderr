error: target enum `NonExistent` is not specified in this enum's #[enum_into] annotation
 --> tests/enum_into/compile_fail/variant/invalid_target_enum.rs:6:25
  |
6 |     #[enum_into(Target, NonExistent)] // Invalid target enum
  |                         ^^^^^^^^^^^
